layouts/index.html

```diff
@@ -9,6 +9,7 @@
 
 
 
+<<<<<<< HEAD
```

Comment:
@femmebot Looks like some merge syntax got in here, and I'm not quite sure how to fix it...

Reply:
(I tried just removing the merge syntax locally, but the homepage displayed weird after I did that.)

Reply:
Ah nevermind, I compared it to https://landing-home.app.cloud.gov/ and figured out how to fix it! Time to merge!